### DISCUSSION TITLE
German display language?

### DISCUSSION THREAD

**Discussion by SR7673** [tbnobody/OpenDTU#904] [Ideas]
So, ich schalt' gleich mal um auf Deutsch...

Wäre es möglich die Sprache im Display auf Deutsch umzuschalten, bzw. die Sprache über die web-app einstellbar zu machen?
Oder gibt es das schon und ich habe es übersehen?

Meine Eltern sind 80+ und können mit "today" nix anfangen, würden sich aber sicher über einen regelmäßigen Blick auf die momentane Erzeugung freuen...

**Comment by DPO99** [comment]
#955
